packages/iptables/iptables.spec

```diff
@@ -71,6 +71,8 @@ sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
 %{_cross_libdir}/*.so.*
 %dir %{_cross_libdir}/xtables
 %{_cross_libdir}/xtables/*.so
+%dir %{_cross_datarootdir}/xtables
+%{_cross_datarootdir}/xtables/iptables.xslt
```

Comment:
nit: `%{_cross_datarootdir}` should be the same as `%{_cross_datadir}`

Seems like we should be able to exclude the xslt file also. It looks to be showing up now because of this commit:
https://git.netfilter.org/iptables/commit/?id=3822a992bc2772d

(exclude can go below the `pf.os` exclude since the directory is the same)